### ISSUE TITLE
Could not build Objective-C module 'Firebase' and 'FirebaseCore/FirebaseCore.h' file not found

### ISSUE THREAD

**Issue by yunusdtarla** [firebase/quickstart-ios#1356]
hi

Xcode>Runner.xcodeproj>Product>Archive i get these errors when I do:

Could not build Objective-C module 'Firebase'
No such module 'Flutter'
'FirebaseCore/FirebaseCore.h' file not found
Note1 = VS Code the application opens

**Comment by paulb777** [comment]
Make sure you open the generated Xcode workspace and not the project.

**Comment by yunusdtarla** [comment]
Xcode>_proje>iOS>Runner.xworkspace I get the same error when I open it.

**Comment by paulb777** [comment]
What about after building? The module cache may need to be reset.

**Comment by yunusdtarla** [comment]
not working. There is an error somewhere, but I can't figure it out. Many internet users are getting the same error. However, when I tried, I couldn't figure it out. I said maybe there is updated information

**Comment by morganchen12** [comment]
Can you build the project without Firebase installed?

**Comment by yunusdtarla** [comment]
Yup. I get an error only when firebase is installed. The application opens. But when we archive in xcode, I get an error:

- Cannot find 'FirebaseApp' in scope
- Could not build Objective-C module 'Firebase'
- 'FirebaseCore/FirebaseCore.h' file not found

Padlock I delete the files and upload them again, but the errors do not change.

**Comment by yunusdtarla** [comment]
![Ekran Resmi 2022-03-03 19 25 55](https://user-images.githubusercontent.com/72821593/156607320-e5bb4665-13d1-4aff-849c-7d318372fea3.png)

**Comment by morganchen12** [comment]
@yunusdtarla was your build able to locate the Firebase headers when you manually added them to the header search paths?
